### ISSUE TITLE
[victoria-metrics-k8s-stack] Type mismatch `($spec.extraArgs).tls` after refactoring

### ISSUE THREAD

**Issue by psmolkin** [VictoriaMetrics/helm-charts#1449]
Hello,
Unfortunately, the last refactoring resulted in a type mismatch error:
https://github.com/VictoriaMetrics/helm-charts/blame/victoria-metrics-k8s-stack-0.25.14/charts/victoria-metrics-k8s-stack/templates/_helpers.tpl#L133-L135
```yaml
# ./victoria-metrics-k8s-stack/templates/_helpers.tpl:133, {{- define "vm.url" -}} 
    {{- $isSecure = ($spec.extraArgs).tls | default $isSecure -}}
    {{- $proto = (ternary "https" "http" $isSecure) -}}
```
This part references the wrong type, since the `extraArgs` field must `object (keys:string, values:string)`:
see [extraArgs](https://docs.victoriametrics.com/operator/api/index.html#vminsert) at `VMClusterSpec.VMInsert`

This results in two mutually exclusive situations:
- String param
  ```yaml
  vmcluster:
    enabled: false
    spec:
      vminsert:
        extraArgs:
          tls: "true"
  ```
   leads to `Error: UPGRADE FAILED: template: victoria-metrics-k8s-stack/templates/victoria-metrics-operator/vmagent/vmagent.yaml:13:9: executing "victoria-metrics-k8s-stack/templates/victoria-metrics-operator/vmagent/vmagent.yaml" at <include "vm.agent.spec" .>: error calling include: template: victoria-metrics-k8s-stack/templates/_helpers.tpl:279:16: executing "vm.agent.spec" at <include "vm.agent.remote.write" .>: error calling include: template: victoria-metrics-k8s-stack/templates/_helpers.tpl:271:56: executing "vm.agent.remote.write" at <include "vm.write.endpoint" $ctx>: error calling include: template: victoria-metrics-k8s-stack/templates/_helpers.tpl:170:37: executing "vm.write.endpoint" at <include "vm.url" $ctx>: error calling include: template: victoria-metrics-k8s-stack/templates/_helpers.tpl:134:41: executing "vm.url" at <$isSecure>: wrong type for value; expected bool; got string`
- Bool param
  ```yaml
  vmcluster:
    enabled: false
    spec:
      vminsert:
        extraArgs:
          tls: true
  ```
    leads to `Error: UPGRADE FAILED: release monitoring failed, and has been rolled back due to atomic being set: cannot patch "monitoring-victoria-metrics-k8s-stack" with kind VMCluster: VMCluster.operator.victoriametrics.com "monitoring-victoria-metrics-k8s-stack" is invalid: spec.vminsert.extraArgs.tls: Invalid value: "boolean": spec.vminsert.extraArgs.tls in body must be of type string: "boolean"`

**Comment by AndrewChubatiuk** [comment]
thanks for an issue, submitted a PR with a fix
all CRs require string arguments, so string value will be expected for `tls` arg value

**Comment by AndrewChubatiuk** [comment]
you can find a fix in [0.25.15](https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-k8s-stack-0.25.15)
